spec/features/user/user_index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
```

Comment:
- [ ] You should have a test case that asserts you can see the profile picture for each user.

Reply:
Done ✔